webpack/components/SelectOrg/SetOrganization.js

```diff
@@ -0,0 +1,109 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Form, Button } from 'patternfly-react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import Select from '../../move_to_pf/Select/Select';
+import * as SelectOrgActions from './SelectOrgAction';
+import reducer from './SelectOrgReducer';
+import { LoadingState } from '../../move_to_pf/LoadingState';
+import './SelectOrg.scss';
+
+class SetOrganization extends Component {
+  constructor(props) {
+    super(props);
+    this.onSelect = this.onSelect.bind(this);
+    this.onSend = this.onSend.bind(this);
+    this.state = { disabled: true };
+  }
+
+  componentDidMount() {
```

Comment:
It's not the perfect location, but it works. 
I'd like to sync react-router with redux , like [this one](https://github.com/supasate/connected-react-router).